sdk/data/aztables/service_client.go

```diff
@@ -106,15 +107,15 @@ func (t *ServiceClient) CreateTable(ctx context.Context, name string, options *C
 }
 
 // Options for Client.Delete and ServiceClient.DeleteTable methods
-type DeleteTableOptions struct {
-}
+type DeleteTableOptions struct{}
```

Comment:
In generated code, we add a comment indicating this is for future expansion.  See [example](https://github.com/Azure/autorest.go/blob/track2/test/autorest/additionalpropsgroup/zz_generated_models.go#L96-L98)

Reply:
Added